polaris.shopify.com/content/components/images-and-icons/icon.mdx

```diff
@@ -23,6 +23,9 @@ examples:
   - fileName: icon-with-custom-svg-and-color.tsx
     title: With custom SVG and color
     description: When using changing color of an svg and it uses currentColor, the white color is applied.
+  - fileName: icon-text-tone-inherit.tsx
+    title: Inherit text tone
+    description: Inherit `tone` of the parent `Text` component
```

Comment:
I wouldn't get too hung up on the phrasing in our documentation right now. I don't see this as a blocker as it's easy to adjust as needed. I would ship what makes sense now then we can revisit these docs when we apply tone following the shared API decisions across components.

Reply:
Agree we can adjust the wording later on this. That being said I'll throw out some thoughts:

- I am wondering if we can combine "Colored" and "Inherit" in one and just call it "Tone". 
- Within "Tone" can we say something like: 
  - Set the icon color using the `tone` property.
  - When an icon is placed inside of a [Text](link to component) component, it will inherit the Text components tone.

That needs some massaging but then you could show examples of both. Tone on the Icon component and Icon wrapped in Text with tone set on Text.

Reply:
Oh I just realized it looks like `inherit` still needs to be set on the icon's "tone". I was thinking it just automatically inherited the tone if the icon was inside text. 

I think this makes even more sense in this case to just group everything under "tone". 

The description might be something like:

Apply a color to the icon color using the `tone` property. Additionally, when an icon is wrapped inside of the `Text` component, use `tone="inherit"` to allow the icon to inherit the Text component's tone.